src/Core/Resources/SharedResources.en.resx

```diff
@@ -652,4 +652,25 @@
     <value>Expected authentication context class reference (acr) was not returned with the authentication response or is invalid.</value>
     <comment>'acr' is an explicit OIDC claim type, see https://openid.net/specs/openid-connect-core-1_0.html#rfc.section.2 (acr). It should not be translated.</comment>
   </data>
+  <data name="ResetPassword" xml:space="preserve">
+    <value>Master Password Reset</value>
+  </data>
+  <data name="ResetPasswordDescription" xml:space="preserve">
+    <value>Allow administrators in the organization to reset master password for users.</value>
```

Comment:
The way "reset master password" is written hear feels grammatically incorrect.
```suggestion
    <value>Allow administrators in the organization to reset organization user's master passwords.</value>
```

Reply:
Hmm. Should it be `...reset organization users' master password` to signify multiple users within the org but each only has a single master password (?)

Reply:
Updated.